examples/ByUrl.purs

```diff
@@ -108,6 +109,7 @@ examples =
   , "MintsMultipleTokens" /\ MintsMultipleTokens.contract
   , "OneShotMinting" /\ OneShotMinting.contract
   , "OneShotMintingV2" /\ OneShotMintingV2.contract
+  , "AwaitTxConfirmedWithTimeout" /\ AwaitTxConfirmedWithTimeout.contract
```

Comment:
Please remove it from here. We use ByURL for E2E tests. There's no need to test it in the browser context, because we can rely on assumption that `Aff` works the same in node (plutip test) and the browser.

Reply:
That's good to know, I wasn't very sure of which things should be convered by E2E